cli/src/clojure_lsp/nrepl.clj

```diff
@@ -12,10 +13,12 @@
 (defn ^:private repl-port []
   (:port (start-server :handler cider-nrepl-handler)))
 
-(defn setup-nrepl [db]
+(defn setup-nrepl [db*]
   (try
     (when-let [port (repl-port)]
       (logger/info "====== LSP nrepl server started on port" port)
-      (swap! db assoc :port port))
+      ;; db/db* only available in dev. Don't use it otherwise.
+      (alter-var-root #'db/db* (constantly db*))
+      (swap! db/db* assoc :port port))
```

Comment:
I don't understand this change, could you elaborate?

Reply:
I see, I think we could be straightforward here and just have a `setup-dev-environment` for now.